packages/datadog-plugin-mocha/src/index.js

```diff
@@ -62,8 +63,12 @@ function createWrapRunTest (tracer, testEnvironmentMetadata, sourceRoot) {
           const activeSpan = tracer.scope().active()
           let result
           try {
-            result = await specFunction()
-            activeSpan.setTag(TEST_STATUS, 'pass')
+            result = await specFunction.call(context)
```

Comment:
I dropped the ball on this one. The spec function was not being called with the same context (for reference, how it is done in mocha: https://github.com/mochajs/mocha/blob/c667d1060f97da24abf48e41a284e037f0b410a0/lib/runnable.js#L366). Now it should be.